tomcat-conf/Catalina/localhost/prweb.xml

```diff
@@ -32,5 +32,12 @@
   <Environment name="prconfig/session/ha/enabled" value="true" type="java.lang.String"/>
   <Environment name="prconfig/dsm/services/stream/pyBaseLogPath" value="/opt/pega/kafkadata" type="java.lang.String" />
   <Environment name="prconfig/cloud/isNodeAgnosticAdminStudio" value="true" type="java.lang.String"/>
+
+  {{ if isTrue .Env.HZ_CLIENT_MODE }}
+  <Environment name="prconfig/cluster/clientserver/clientmode" value="{{ .Env.HZ_CLIENT_MODE }}" type="java.lang.String"/>
+  <Environment name="prconfig/cluster/clientserver/hzOnK8s" value="{{ .Env.HZ_DISCOVERY_K8S }}" type="java.lang.String"/>
+  <Environment name="prconfig/identification/cluster/name" value="{{ .Env.HZ_CLUSTER_NAME }}" type="java.lang.String"/>
+  <Environment name="prconfig/cluster/clientserver/server/dns" value="{{ .Env.HZ_DNS }}" type="java.lang.String"/>
```

Comment:
Do you really need the address of the DNS server?  Or is this the URL of the Hazelcast server?

Reply:
I'd also like to understand what this HZ_DNS env var is used for.

Reply:
We have decided to go with this name `HZ_SERVER_URL` .
It will be used for kubernetes based discovery between hazelcast clients and servers.
https://github.com/hazelcast/hazelcast-kubernetes#dns-lookup